### PR TITLE
Change pool

### PR DESCRIPTION
**Pull request by kimpers**
**Desktop**
image
image

**Mobile**
image
image

The UX on mobile isn't great where you have to scroll to the bottom of the page to confirm changing pools. However, I think it can work for now.

### CODE REVIEW

File: ts/components/staking/thumbnail.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+interface ThumbnailProps {
+    size: number;
+    poolId: string;
+    address: string;
+    thumbnailUrl?: string;
+    className?: string;
+}
+
+export const Thumbnail: FC<ThumbnailProps> = ({ thumbnailUrl, size, poolId, address, className }) =>
```
Comment (kimpers) [inline ts/components/staking/thumbnail.tsx]: TODO: refactor to use this in all places

File: ts/components/staking/change_pool_dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    onDismiss,
+    stakingPools,
+    moveStake,
+    currentPoolDetails = {},
+}) => {
+    const { poolId: fromPoolId, zrxAmount } = currentPoolDetails;
+
+    const [search, setSearch] = useState<string>('');
+    const [poolsList, setPoolsList] = useState<PoolWithStats[]>([]);
+    const [fuse, setFuse] = useState<Fuse<PoolWithDisplayName, { keys: string[] }> | undefined>(undefined);
```
Comment (kimpers) [inline ts/components/staking/change_pool_dialog.tsx]: **TODO:** move to re-usable hook

**Review by johnrjj — APPROVED** [review]
👏 👏 👏 👏 👏  great work!

**Comment by johnrjj** [comment]
Looks great!

**Pull request merged**